README.md

```diff
@@ -199,6 +199,35 @@ return static function (ContainerConfigurator $containerConfigurator): void {
 };
 ```
 
+### Configuring Rectors
+
+Every rector can have its own configuration. E.g. the `DowngradeTypedPropertyRector` rule will add a docblock or not depending on its property `ADD_DOC_BLOCK`:
+
+```php
+<?php
+// rector.php
+
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Rector\Downgrade\Rector\Property\DowngradeTypedPropertyRector;
```

Comment:
please sort use statements

Reply:
It would be useful to have a coding standard tool to check this in all markdown files:

https://github.com/symplify/symplify/issues/2055